portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,6 +27,7 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    comments = new ArrayList<>();
     response.setContentType("text/html;");
```

Comment:
Interesting that we can include the ";" in the content type. I guess that supporting having multiple content types